### ISSUE TITLE
rtfobj ole package inconsistency in Python 2 and Python 3

### ISSUE THREAD

**Issue by jshlbrd** [decalage2/oletools#280]
I'm observing some inconsistent behavior between Python 2 and 3 when rtfobj attempts to identify an OLE package (Python 2 works as expected, Python 3 does not). This error causes rtfobj to fail to correctly extract files when run with Python 3. You can see the issue here:

Python 2
```
>>> with open("/path/to/file.rtf","rb") as fin:
...   rtf = rtfobj.RtfObjParser(fin.read())
... 
>>> rtf.parse()
>>> for obj in rtf.objects:
...   print(obj.is_ole)
...   print(obj.is_package)
... 
True
True
True
True
True
True
True
True
False
False
False
False
False
False
```

Python 3
```
>>> with open("/path/to/file.rtf","rb") as fin:
...   rtf = rtfobj.RtfObjParser(fin.read())
... 
>>> rtf.parse()
>>> for obj in rtf.objects:
...   print(obj.is_ole)
...   print(obj.is_package)
... 
True
False
True
False
True
False
True
False
False
False
False
False
False
False
```

**Comment by decalage2** [comment]
Hi Josh, could you please share the RTF sample, so that I can reproduce the bug?

**Comment by jshlbrd** [comment]
Sure, the SHA256 hash for the sample I'm using is 95416dd64701ca61ac4543b31bc1337007d0d568cb07466c30db2e49fde84f99. It's on a few popular malware sharing websites. If you don't have access to any of them, then I can send it to you directly.

**Comment by jshlbrd** [comment]
I may have found the problem: https://github.com/decalage2/oletools/blob/master/oletools/rtfobj.py#L667

Changing this to:
```
                if obj.class_name.lower() == 'package':
```

Resolves the issue for me.

**Comment by decalage2** [comment]
Fixed with commit 9914890
